services/garage-backend/src/services/userService.ts

```diff
@@ -50,3 +52,48 @@ export const upsertUser = async (
 
   return user;
 };
+
+function findUserById(userId: string): Promise<UserDocument> {
+  return User.findById(userId)
+    .exec()
+    .then((user) => {
+      if (!user) {
+        throw new Error(`User ${userId} not found`);
+      }
+      return user;
+    });
+}
+
+function findAllUser() {
+  return User.find().sort({ fullName: 1, email: -1 }).exec();
```

Comment:
```suggestion
  return User.find().exec();
```
sorting breaks our deployed versions